### ISSUE TITLE
Unable to run...

### ISSUE THREAD

**Issue by shaochee** [Katanaco/Schmitt_CSCI2270_FinalProject#1]
I was unable to run through the terminal and received errors about undeclared functions. It would not run in codeBlocks due to a seg fault/the hash table did not exist. Would not build or run in either case...

**Comment by Katanaco** [comment]
When compiling via command line did you include the main, hastable.cpp, and hashtable.h?

**Comment by Katanaco** [comment]
My apologies I mean main.cpp, showTraker.cpp, and showTraker.h

**Comment by shaochee** [comment]
Yes, I tried that! I'm not the greatest at computer science so it might have been user error on my part. I tried everything I knew, but I couldn't get your code to run.

**Comment by Katanaco** [comment]
Are you running on the Ubuntu vm or another Operating system?

**Comment by shaochee** [comment]
I am running it on the CSCI2700 Virtual Machine!

-----Original Message-----

**Comment by Katanaco** [comment]
That is odd, that's what we've been developing the project on; give me a minute to check that it has all of the proper directing and dependencies.

**Comment by Katanaco** [comment]
To check that the executable functions on yours could you try ./a.out? I am almost certain it will but I'd still like to check

**Comment by DerekRhodes** [comment]
Hi guys, I got the code to run, however I am getting many -1 values when I select Print Contents option. I am trying to debug now.

**Comment by DerekRhodes** [comment]
I also get an error when I select quit. I see that some of the -1 values are from the text file, so that might be intentional.
